ConvertTo-Jpeg.ps1

```diff
@@ -85,9 +92,21 @@ Process
                 continue
             }
             $bitmap = AwaitOperation ($decoder.GetSoftwareBitmapAsync()) ([Windows.Graphics.Imaging.SoftwareBitmap])
+            
+            # Determine output file name (whether to keep or remove existing extension)
+            if ($RemoveExistingExtension)
+            {
+                # Name new file same as old file, except change extension to jpg
+                $extension = $inputFile.FileType
+                $outputFileName = $inputFile.Name -replace ($extension + "$"), ".jpg"
+            }
+            else
+            {
+                # Name new file same as old file, but add .jpg to the end
+                $outputFileName = $inputFile.Name + ".jpg";
```

Comment:
Instead of appending .jpg in two places, let's get rid of the else clause, always run this line, and not add .jpg above.

Reply:
I see two ways to do this, one using the ternary operator:
```Powershell
$outputFileName = $(If ($RemoveOriginalExtension) {$inputFile.DisplayName} Else {$inputFile.Name}) + ".jpg"
```
and one using an additional variable:
```Powershell
$fileName = $inputFile.Name
if ($RemoveOriginalExtension)
{
    $fileName = $inputFile.DisplayName 
}
$outputFileName = $fileName + ".jpg"
```
Which do you prefer?

Reply:
Assuming `DisplayName` represents the name without the extension, the second example you show (plus the nice comments you have) seems good to me!